src/NServiceBus.Core/DataBus/DataBusSendBehavior.cs

```diff
@@ -89,7 +89,7 @@ public class Registration : RegisterStep
         {
             public Registration(Conventions conventions) : base("DataBusSend", typeof(DataBusSendBehavior), "Saves the payload into the shared location", b => new DataBusSendBehavior(b.Build<IDataBus>(), b.Build<IDataBusSerializer>(), conventions))
             {
-                InsertAfter(WellKnownStep.MutateOutgoingMessages);
+                InsertAfter("MutateOutgoingMessages");
```

Comment:
I was thinking the same. mutators are the only reason I can come up with where a before/after makes some sense.